pkg/kubelet/logs/container_log_manager.go

```diff
@@ -213,80 +223,121 @@ func (c *containerLogManager) Clean(ctx context.Context, containerID string) err
 	return nil
 }
 
+func (c *containerLogManager) processQueueItems(worker int) {
+	klog.V(4).InfoS("Starting container log rotate worker", "workerID", worker)
+	for c.processContainer(context.Background(), worker) {
```

Comment:
Would this read easier if this layer was removed and the loop just ran on `key, quit := c.queue.Get()`?

i.e. (Dunno if it can be iterated like this, but you get the idea).

```
for key, quit := c.queue.Get() {
    // Do stuff
}
```

Reply:
The function based call seem to be the pattern for most of the controllers that I have seen on the k8s side. I am open to change if there is an advantage of doing that over what I have now.

Reply:
The advantage of using the `for function() {}` Is, it makes it easy to terminate the loop by sending a `false` from the function.